add-steps/haproxy.cfg

```diff
@@ -0,0 +1,26 @@
+defaults
+    mode    http
+    balance roundrobin
```

Comment:
I'll try that out. I am reluctant to make that the final config because I don't want it to be HAProxy vs. linkerd. If that were the goal, I wouldn't put HAProxy in the linkerd cluser's request path.